receiver/splunkhecreceiver/receiver.go

```diff
@@ -432,10 +432,10 @@ func (r *splunkReceiver) validateChannelHeader(channelID string) error {
 
 func (r *splunkReceiver) handleReq(resp http.ResponseWriter, req *http.Request) {
 	ctx := req.Context()
-	if r.logsConsumer != nil {
+	if r.logsConsumer != nil && r.metricsConsumer == nil {
```

Comment:
Isn't this always the case given that `splunkReceiver` instance is never shared between metrics and tracing pipeline?

Reply:
Nevermind. I missed that the receiver uses the shared component library